tests/fixtures.py

```diff
@@ -22,27 +22,24 @@
     "shared": {
         "a_valid_reference": VALID_REF,
         "an_invalid_reference": NON_REF,
-        "a_malformed_reference": MALFORMED_REF,
     },
     "Test_identifier": {"test_matches_fixture": {"identifier": IDENTIFIER}},
     "Test_displayName": {"test_matches_fixture": {"display_name": "My Asset Manager"}},
     "Test_info": {
         "test_matches_fixture": {
-            "info": {kField_EntityReferencesMatchPrefix: "my_asset_manager:///"}
+            "info": {constants.kField_EntityReferencesMatchPrefix: "my_asset_manager:///"}
```

Comment:
[fix] Needs updating to the new name as this one is deprecated.

Reply:
Yep, changed in [801d7c6](https://github.com/elliotcmorris/Template-OpenAssetIO-Manager-Python/pull/3/commits/801d7c67ccf11bf3cc09d0244ef6fd9d9fc78e92)

Reply:
[fix] Needs updating [here](https://github.com/elliotcmorris/Template-OpenAssetIO-Manager-Python/blob/3dc699370206228b16e58a2463ca59611988bcab/plugin/my_asset_manager/MyAssetManagerInterface.py#L62) too

Reply:
:anger: :man_facepalming:  https://github.com/elliotcmorris/Template-OpenAssetIO-Manager-Python/pull/3/commits/ae842b0aabf258cff551eb38922d09410f5dacec